addons/hr_holidays/models/hr_leave_accrual_plan.py

```diff
@@ -3,6 +3,11 @@
 
 from odoo import api, fields, models, _
 
+DAY_SELECT_VALUES = [str(i) for i in range(1, 29)] + ['last']
+DAY_SELECT_SELECTION_NO_LAST = tuple(zip(DAY_SELECT_VALUES, (str(i) for i in range(1, 29))))
+
+def _get_selection_days(self):
+    return DAY_SELECT_SELECTION_NO_LAST + (("last", _("last day")),)
```

Comment:
I feel like this is an overly complex solution. Maybe we can just use a datepicker and ignore the year?

Reply:
Although that's also not ideal...

Reply:
I don't want to rewrite this for now, it's not in the spec either.

Reply:
Fair enough